assets/stylesheets/application.scss

```diff
@@ -1,3 +1,7 @@
+@import "tailwindcss/base";
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
+
 body {
```

Comment:
Should we move the base style into another file?

Reply:
@hoangmirs I just did a simple things, I'll remove these once I'm working on feature tickets ✌️